recipes/conversation_view/message_input/message_input.vue

```diff
@@ -145,13 +145,24 @@
         <!-- Right content -->
         <div class="d-d-flex">
           <!-- Optionally displayed remaining character counter -->
-          <p
+          <dt-tooltip
             v-if="displayCharacterLimitWarning"
+            :enabled="characterLimitWarningMessage && (characterLimitCount - inputLength < 0)"
+            placement="top-end"
+            :message="characterLimitWarningMessage"
             class="d-fc-error d-mr16 d-as-center dt-message-input--remaining-char"
```

Comment:
Should these classes be on the anchor <p> instead of the tooltip itself?

Reply:
i moved everything from <p> as i had an issue with it showing up. didnt have to move classes. Moved it back.